### ISSUE TITLE
[Feature Request] Add actions for OS 4 multitasking

### ISSUE THREAD

**Issue by PrabhuDeva** [rpetrich/libactivator#50]
Well, title basically says it all: With OS 4 and Apple's new multitasking, we need a new action to activate the task switcher (personally, I can't stand using double home button press, as it can be inaccurate). Hope you'll have the time to implement this once Activator is working 100% on OS4 again.

Also, an interesting thing would be a new system action of completely closing an app (even if it has built-in multitasking) and thus also preventing it from showing in task switcher until launched again from Springboard. This way, people could use Apple multitasking much more selectively, just as they use Backgrounder now.

**Comment by rpetrich** [comment]
Added in 1.4.0 beta 2

**Comment by PrabhuDeva** [comment]
Great, thank you.

However, I found that not all activation methods are able to activate the switcher: I was not able to use any of the three "slide in from bottom" gestures, the "hold status bar" gesture, the pinch/spread springboard gestures to activate it. As far as I tested it, the other activation methods do work fine for activating the switcher. I tested most of these on the home screen only, but I think it applies also to "in application" activating.
